web/gds-user-ui/src/modules/dashboard/certificate/lib/trixoQuestionnaireValidationSchema.ts

```diff
@@ -14,16 +14,15 @@ export const trixoQuestionnaireValidationSchema = yup.object().shape({
     has_required_regulatory_program: yup.string().oneOf(['no', 'yes']).default('no'),
     conducts_customer_kyc: yup.boolean().default(false),
     kyc_threshold: yup
-      .number()
+      .string()
       .transform((value, originalValue) => {
```

Comment:
I attempted to make the originalValue type a string `(originalValue: string)` here, but that didn't resolve the issue.